devito/logger.py

```diff
@@ -13,13 +13,13 @@
 stream_handler = logging.StreamHandler()
 logger.addHandler(stream_handler)
 
-# Add extra logging levels (note: INFO has value=20, WARNING has value=30)
-DEBUG = logging.DEBUG
+# Add extra logging levels
+DEBUG = logging.DEBUG  # value=10
```

Comment:
implementation detail

Reply:
Not sure I understand what to change here

Reply:
I think the numbers should be dropped

Reply:
I am reverting as before, some detail is useful